tests/foreman/ui/test_navigation.py

```diff
@@ -0,0 +1,115 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Login UI
+
+@Requirement: Navigation
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: UI
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+from robottelo.decorators import skip_if_bug_open, tier1
+from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators, menu_locators
+from robottelo.ui.session import Session
+
+
+class NavigationTestCase(UITestCase):
+    """Implements the navigation tests from UI"""
+
+    def page_objects(self):
+        return {
+            'Activation Key': self.activationkey,
```

Comment:
>I'm fine with keeping the dict in here, but why method instead of dict?
Because, I need to have instance(e.g. 'self') to work with and I didn't find any other way to perform that